packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -39,57 +46,54 @@ export const getSplitButtonVerticalDividerClassNames = memoizeFunction(
   }
 );
 
-export const getContextualMenuClassNames = memoizeFunction(
-  (theme: ITheme, className?: string): IContextualMenuClassNames => {
-    const styles = getContextualMenuStyles(theme);
-
-    return mergeStyleSets({
-      container: [
-        'ms-ContextualMenu-container',
-        styles.container,
-        className,
-        [
-          {
-            selectors: {
-              ':focus': { outline: 0 }
-            }
-          }
-        ]
-      ],
-      root: ['ms-ContextualMenu is-open', styles.root],
-      list: ['ms-ContextualMenu-list is-open', styles.list],
-      header: ['ms-ContextualMenu-header', styles.header],
-      title: styles.title
-    });
-  }
-);
+const GlobalClassNames = {
+  item: 'ms-ContextualMenu-item',
+  divider: 'ms-ContextualMenu-divider',
+  root: 'ms-ContextualMenu-link',
+  isChecked: 'is-checked',
+  isExpanded: 'is-expanded',
+  isDisabled: 'is-disabled',
+  linkContent: 'ms-ContextualMenu-linkContent',
+  linkContentMenu: 'ms-ContextualMenu-linkContent',
+  icon: 'ms-ContextualMenu-icon',
+  iconColor: 'ms-ContextualMenu-iconColor',
+  checkmarkIcon: 'ms-ContextualMenu-checkmarkIcon',
+  subMenuIcon: 'ms-ContextualMenu-submenuIcon',
+  label: 'ms-ContextualMenu-itemText',
+  secondaryText: 'ms-ContextualMenu-secondaryText'
+};
 
+// TODO: Audit perf. impact of and potentially remove memoizeFunction.
+//       https://github.com/OfficeDev/office-ui-fabric-react/issues/5534
 export const getItemClassNames = memoizeFunction(
```

Comment:
I don't think memoize is going to work here. The input will constantly mutate, breaking the point.

Reply:
👍 will remove in subsequent PR

Reply:
You called it #6738.